yoti-sdk/src/main/java/com/yoti/mobile/android/sdk/YotiSDKButton.java

```diff
@@ -70,6 +70,10 @@ public void init(AttributeSet attrs) {
         setOnClickListener(this);
     }
 
+    public void setUseCaseId(String useCaseId) {
+        mUseCaseId = useCaseId;
+    }
+    
     private void setLeftRigthPadding(){
```

Comment:
@eduardoyoti There's a typo here. "Right" is wrongly spelt. Should have been `setLeftRightPadding`.